components/LayerObject.vue

```diff
@@ -10,6 +10,10 @@
 
     let ul;
     let selection = ref(undefined);
+    let navHeight = ref(0);
+    let navHeightPx = computed(() => {
+        return navHeight.value + 'px';
+    });
```

Comment:
This should rather be handled over a restructuring of the pop-up component than with this weird logic of addin margin to the first layer.

Reply:
Will do in next brach